scripts/Makefile

```diff
@@ -141,20 +141,36 @@ cluster-deprovision:
 #
 .PHONY: nodes-deprovision
 nodes-deprovision:
-	$(MAKE) echo-nodes-deprovision
 	$(MAKE) fix-s3-backend
-	$(MAKE) clean-state
-	$(MAKE) sync-files
-	$(MAKE) terraform-init
-	$(MAKE) terraform-remove-instance
+	SHELL=/bin/bash
+
+	# Find the existing node
```

Comment:
can you put this to bash or python script instead - this will be hard to modify/debug as a Makfile target tbh